www/src/components/showcase-details.js

```diff
@@ -155,7 +153,7 @@ function ShowcaseModal({ children, location }) {
       modalNextLink={
         <>
           <Img
-            key={nextSite.id}
+            key={nextSite.fields.slug}
```

Comment:
are these keys even needed? They're not part of a list or anything.

Reply:
Good q … l👀king now (like I should have before I approved 😬).

Reply:
I think keys are there to force components to unmount when cycling through showcase with prev/next buttons and without them there was some weirdness (old image staying visible or something like that)

Reply:
@pieh to the rescue! 💖

Reply:
After thinking a bit more - this just seem like problem with `gatsby-image` that doesn't nicely handle case of props updates (i.e. it should blow away whatever state it had)? (if it still is the case, not sure if it is).